crates/bevy_time/src/lib.rs

```diff
@@ -68,7 +73,14 @@ impl Plugin for TimePlugin {
 
         app.add_systems(
             First,
-            time_system
+            (time_system, queue_timed_commands::<Virtual>)
```

Comment:
Just to know
So if I use some kind of custom Timer, this will only work if I add the queue system to the said timer?